src/Worker.cs

```diff
@@ -101,6 +105,15 @@ public async static Task Main(string[] args)
             await requestProcessor.ProcessRequestLoop();
         }
 
+        private static void SetVersionUpgradeOptOut()
+        {
+            string checkForUpgrade = Environment.GetEnvironmentVariable("AZUREPS_CHECK_FOR_UPGRADE");
+            if (string.IsNullOrEmpty(checkForUpgrade))
```

Comment:
Please use `string.IsNullOrWhiteSpace` instead of `string.IsNullOrEmpty` to validate for null, empty or white space.